package-lock.json

```diff
@@ -158,6 +158,11 @@
         "color-convert": "1.9.1"
       }
     },
+    "any-observable": {
+      "version": "0.2.0",
+      "resolved": "https://registry.npmjs.org/any-observable/-/any-observable-0.2.0.tgz",
+      "integrity": "sha1-xnhwBYADV5AJCD9UrAq6+1wz0kI="
+    },
```

Comment:
What are all these changes?

Reply:
Those are those ide files that I'm waiting to remove. Don't worry about them; I'm going to push up a commit afterwards to get rid of them. I just didn't want all those removals further obfuscating the PR.